.circleci/config.yml

```diff
@@ -0,0 +1,46 @@
+# Javascript Node CircleCI 2.0 configuration file
```

Comment:
@gorakong You have a CircleCI configuration file, but I don't see any results for passing or failed linting checks in the PR. Is your CircleCI set up correctly?

Reply:
@gorakong Hey Gora, congrats on getting that set up. You might want to look at having CircleCI run your linter as well prior to merging.